service/controller/clusterapi/v31/resource/tccpingresstargetgroups/create.go

```diff
@@ -0,0 +1,86 @@
+package tccpingresstargetgroups
```

Comment:
I am not sure what your reasoning was but we have already a resource to fetch the TCCP stack outputs which are then put into the controller context. See https://github.com/giantswarm/aws-operator/blob/master/service/controller/clusterapi/v31/resource/tccpoutputs/create.go. I think this can simply be extended so we do not need to have another resource for it.

Reply:
The problem here was that the resource also needs to be used in machineDeployment controller and this value was not propagated, I previously had it in this resource and it was nil at the place where I need to use it.

Reply:
I see. You want the TCCP outputs in the MachineDeployment controller. Then I would configure the `toCluster` function in the `tccpoutputs` resource, extend it with the two new values and wire it in the MachineDeployment controller. This would be more straight forward because we then only have one resource each for all the outputs. I feel like having a different resource increases the complexity of the approach and we do not need to do this.